js/audio/Index.svelte

```diff
@@ -212,10 +194,10 @@
 			waveform_options={gradio.props.waveform_options}
 			{trim_region_settings}
 			stream_every={gradio.props.stream_every}
-			bind:modify_stream={_modify_stream}
-			bind:set_time_limit
+			stream_state={gradio.shared.loading_status.stream_state}
 			upload={(...args) => gradio.shared.client.upload(...args)}
 			stream_handler={(...args) => gradio.shared.client.stream(...args)}
+			time_limit={gradio.shared.loading_status.time_limit}
```

Comment:
`pending` is now inferred from the loading status. The LoadingStatus now contains a `type` prop of either 'input' or 'output' so components can decide what to do.

`stream_state` and `time_limit` are now tracked on the LoadingStatus as well, and updated along wit status updates, so we don't need to do any gymnastics to make it work.